apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -25,6 +25,32 @@ export default function RubricsContainer({
             learningGoal: '',
             aiEnabled: false,
             position: 1,
+            learningGoalEvidenceLevelsAttributes: [
+              {
+                learningGoalId: 'learningGoal-1',
```

Comment:
do you need `learningGoalId` here?

Reply:
It is there in part because that is needed to save the `learningGoalEvidenceLevels`.  That being said, as I have explored fixing the edit functionality, I am having second thoughts about creating these Ids before saving anything.

Reply:
Is it needed to save `learningGoalEvidenceLevels`? It's not a permitted param in `rubrics_controller`

Reply:
> I am having second thoughts about creating these Ids before saving anything.

I am curious if you need these ids at all. They're not the ones saved to the database -- they're not permitted params so MySql is generating them on save. You do need a unique identifier for learning goals to use as a `key` prop though, even before saving 🤔

Reply:
Yes!  That is where I landed too... I think I was having a hard time knowing what a `key` vs an `id` was for in both the front and back end, but I feel like I have it worked out a bit better, AND in the process fixed the editing a rubric process!